main.go

```diff
@@ -218,75 +218,239 @@ func dialectFromArg(arg string) Dialect {
 	return UNKNOWN
 }
 
-func main() {
-	GLOBAL_ENV.FindNamespace(MakeSymbol("user")).ReferAll(GLOBAL_ENV.CoreNamespace)
-	if len(os.Args) == 1 {
-		repl(EVAL)
-		return
-	}
-	if len(os.Args) == 2 {
-		if os.Args[1] == "-v" || os.Args[1] == "--version" {
-			println(VERSION)
-			return
-		}
-		processFile(os.Args[1], EVAL)
-		return
-	}
-	workingDir := ""
-	phase := EVAL
-	lint := false
-	dialect := UNKNOWN
-	expr := ""
-	length := len(os.Args) - 1
-	for i := 1; i < length; i++ {
-		switch os.Args[i] {
+func usage() {
+	fmt.Fprintf(os.Stderr, "Joker - %s\n\n", VERSION)
```

Comment:
I think help should be printed to stdout, not stderr. Would you agree?

Reply:
I've seen both approaches taken, but I do prefer `stdout` actually...I've made it easier to change in the future besides changing it now.